CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/JDBCImportController.java

```diff
@@ -193,20 +193,26 @@ protected void updateSampleData() {
 
         if (configurationPane != null) {
             configurationPane.setSampleData(currentColumns, currentData);
-            openConfigPane(!currentColumns[0].isBlank());
-            enableButton(currentColumns[0].isBlank());
+
+            // If the current columns lenght isn't 0, expand the config pane
+            // and enable the import button
+            if (!currentData.isEmpty()) {
+                openConfigPane(StringUtils.isNotBlank(currentData.get(0).toString()));
+                disableButton(StringUtils.isBlank(currentData.get(0).toString()));
+            } else {
+                openConfigPane(false);
+                disableButton(true);
+            }
```

Comment:
Given `currentData` is a `List<String[]>`, the `toString()` of an array won't ever be blank so effectively the code could be simplified to 
```
openConfigPane(!currentData.isEmpty())
disableButton(currentData.isEmpty())
```

Reply:
Should probably also change the comment to refer to `currentData` rather than `currentColumns`